src/main.cpp

```diff
@@ -599,20 +597,19 @@ int main ( int argc, char** argv )
         bUseGUI = false;
         qWarning() << "No GUI support compiled. Running in headless mode.";
     }
-#endif
 
-    if ( bIsClient )
+#endif
 #ifdef SERVER_ONLY
+    if ( bIsClient )
```

Comment:
Probably the initial idea was to avoid duplicating the if, but It should be ok either way.

Reply:
Yep, I can see why it was done.  I prefer that the code "looks right" without the pre-processor commands, though - there's less problem with IDEs, too.